### ISSUE TITLE
Time and GPU needed 

### ISSUE THREAD

**Issue by CryptoGrogu** [johguse/profanity#49]
HI,

how much time would you estimate for full custom address 40 characters ?
I have 4x2080ti 124gb ram 1tb nvme ssd xeon processor

Thank you =)

**Comment by k06a** [comment]
@CryptoGrogu could you provide MH/s speed of each 2080ti? I will be glad to help you to estimate 😎

**Comment by BarrowHill** [comment]
the problem is, johguse has died 2 years ago and now he cant fix his program. This program can't made cusom address, this program is just doing leading 0, leading f and notjing else. for example you cant create address with 0123456 for example, because there is no such function.

**Comment by k06a** [comment]
@BarrowHill that's not true, you can use the following command to find address with desired prefix:
```
./profanity --matching 000444aaaXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
```

**Comment by SkyTradeInc** [comment]
@k06a can you share your method for estimating time to find a contract address

350 MH/s, how long to find for example 14 leading 0's?
any other insight will be extremely helpful 

cheers!

**Comment by k06a** [comment]
@SkyTradeInc sure. Finding X leading zeroes requires 2^(4*X) operations, 14 leading zeroes is 2^56 ≈ 64 * 10^15 ≈ 64,000,000,000 MH

Then divide MH by the 350 MH/s speed to know number of seconds ≈ 6.5 years

**Comment by salian** [comment]
> how much time would you estimate for full custom address 40 characters ? I have 4x2080ti 124gb ram 1tb nvme ssd xeon processor

Approximately 10^32 years for the full 40 characters on that hardware. 
Currently we are in 2022 so it should be done by the year 10000000000000000000000000002022.
